docs/DEPLOYMENT.md

```diff
@@ -1,7 +1,8 @@
 # Deployment
 
 ## Azure
-Create and set the environment variables based on `.env.template`. Make sure `FINESSE_BACKEND_DEBUG_MODE` is not set to `True`.
+Create and set the environment variables based on `.env.template`. Make sure 
+`FINESSE_BACKEND_DEBUG_MODE` is not set to `True`.
```

Comment:
why?

Reply:
@k-allagbe as well

Reply:
Why the change?: to keep 80 char max lines.
Why that directive: So that we don't deploy in debug mode.

Reply:
it may be that we want to deploy a particular dev wrong in debug mode; I would explain that this should not be set or set to False for production version.

Reply:
Done.